baipw/middleware.py

```diff
@@ -61,11 +61,10 @@ def _basic_auth_response(self, request):
             return self.get_response_class()(request=request)
 
     def _get_client_ip(self, request):
-        function_path = getattr(settings, "BASIC_AUTH_GET_CLIENT_IP_FUNCTION", None)
-        func = get_client_ip
-        if function_path is not None:
-            func = import_string(function_path)
-        return func(request)
+        function_path = getattr(
+            settings, "BASIC_AUTH_GET_CLIENT_IP_FUNCTION", "baipw.utils.get_client_ip"
```

Comment:
Note: It might look weird to hard code a function reference here to our own module. I'm reworking how settings are managed here to help make this clearer, but that'll be a separate PR.

Reply:
I think this should return `None` by default and we should not have any default implementation.